pkgs/development/compilers/bs-platform/bs-platform-62.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchFromGitHub, ninja, nodejs, python35 }:
+let
+  version = "6.2.1";
+  ocaml-version = "4.06.1";
+  src = fetchFromGitHub {
+    owner = "BuckleScript";
+    repo = "bucklescript";
+    rev = "${version}";
+    sha256 = "0zx9nq7cik0c60n3rndqfqy3vdbj5lcrx6zcqcz2d60jjxi1z32y";
+    fetchSubmodules = true;
+  };
+  ocaml =  import ./ocaml.nix {
+    bs-version = version;
+    version = ocaml-version;
+    inherit stdenv;
+    src = "${src}/ocaml";
+  };
+in
+stdenv.mkDerivation {
+  inherit src version;
+  pname = "bs-platform";
+  BS_RELEASE_BUILD = "true";
+  buildInputs = [ nodejs python35 ];
```

Comment:
20.03 will still have python35 but by the time we have 20.09 it reaches EOL and may not be part of the package set. We should avoid using older Python versions.

Reply:
I can try to add some patches around but currently `python3.5` is called in some configure script in bucklescript build.

Reply:
Actually this was explicitely specified in sed happening in `patchPhase`. Now we're using moving target `python3`.